frontend/src/MainBlock/Artifacts/Artifacts.js

```diff
@@ -0,0 +1,51 @@
+import React, { Fragment, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import { withStyles } from '@material-ui/core/styles';
+
+import { fetchArtifactTree } from 'redux/models/Artifact/artifactsActions';
+import { genericPaper } from 'Utils/commonStyles';
+import ArtifactRepositoryItem from './ArtifactRepositoryItem';
+
+const styles = genericPaper;
+
+function Artifacts({ classes, artifactTree, fetchArtifactTree }) {
+    useEffect(() => {
+        fetchArtifactTree();
```

Comment:
чтобы не перегружать сервак, сюда можно добавить проверку
```suggestion
        !artifactTree && fetchArtifactTree();
```